api/server/agent.go

```diff
@@ -32,21 +36,22 @@ func (tapi ToDDApi) Agent(w http.ResponseWriter, r *http.Request) {
 		// Make sure UUID string actually contains something
 		if len(uuid[0]) > 0 {
 			// Let's get the full list so we can identify the right agent if the user specified a short
-			full_agent_list := tdb.DatabasePackage.GetAgents()
+			full_agent_list, _ := tapi.tdb.GetAgents()
```

Comment:
Yeah I definitely need to go back and handle my own errors, so don't worry about existing stuff. If you could just make sure you handle any errors that you've introduced in this patch, that would be great.